packages/ListBox/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import Button from "@paprika/button";
+import PropTypes from "prop-types";
+import { FooterContainerStyled } from "./Footer.styles";
+import useListBox from "../../useListBox";
+import applyOnChange from "../../helpers/applyOnChange";
+
+const propTypes = {
+  isAcceptVisible: PropTypes.bool,
+  isCancelVisible: PropTypes.bool,
+  isClearVisible: PropTypes.bool,
```

Comment:
I thought that as of yesterday we were adopting the single prop pattern...?

Reply:
We will revisit this component when we have access to the `<ActionDialog />` for now I didn't want to go deeper in fixing this as well, I just copy the one that we had previously in our UI library.